inference-engine/src/multi_device/multi_device_exec_network.hpp

```diff
@@ -39,6 +39,26 @@ struct DeviceInformation {
     std::string defaultDeviceID;
 };
 
+struct AutoLoadContext {
+    std::atomic<bool> isEnabled;
+    std::atomic<bool> isAlready;
+    std::atomic<bool> isLoadSuccess;
+    std::future<void> future;
+    std::promise<void> promise;
+    InferenceEngine::SoExecutableNetworkInternal executableNetwork;
+    DeviceInformation  deviceInfo;
+    std::vector<DeviceInformation> metaDevices;
```

Comment:
networkPrecision and metaDevices are used only by _loadContext[Actual] for "selectDevice()",
_loadContext[CPU] didn't use and assign value for them.
add this two members into this Struct is because of 
1. metaDevices(devicelist) is changing in the loading process.  so need to save it 
2. add networkPrecision into struct can reduce this number of parameters of TryToLoadNetWork